fairworkflows/nanopub.py

```diff
@@ -217,16 +217,27 @@ def fetch(uri, format=Format.TRIG):
 
 
     @staticmethod
-    def rdf(assertionrdf, uri=DEFAULT_URI, introduces_concept=None, derived_from=None):
+    def rdf(assertionrdf, uri=DEFAULT_URI, introduces_concept=None, derived_from=None, attributed_to=None, nanopub_author=None):
```

Comment:
It would be nice to document arguments so it makes it easier to follow along.

Reply:
You are right. I'll add some proper documentation.

Reply:
Added.